packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -469,6 +475,17 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          <div className='form-group'>
+            <label>{_('remoteEncryptionKey')}</label>
+            <input
+              className='form-control'
+              name='encryptionKey'
+              onChange={effects.linkState}
+              required
+              type='text'
+              value={encryptionKey}
```

Comment:
Make sure the `value` props is never `undefined` or the input will be uncontrolled.